sbt-sangria-codegen/src/main/scala/com.mediative.sangria.codegen.sbt/SangriaCodegenPlugin.scala

```diff
@@ -63,7 +63,7 @@ import sbt.Keys._
 object SangriaCodegenPlugin extends AutoPlugin {
 
   object autoImport {
-    val SangriaCodegen        = config("sangria-codegen")
+    val SangriaCodegen        = config("sangria-codegen").hide
```

Comment:
It marks the configuration as private. I am not sure if it affects how sbt shows settings scoped by the config.